### PR TITLE
feat: implement simple express api with nodemailer

### LINKED ISSUE #2: Implement backend for verifying @pse.dev emails

### PR DESCRIPTION
**Pull request by ctrlc03**
Add a simple API service which sends and validate OTPs

### CODE REVIEW

**Comment by AtHeartEngineer** [comment]
I can't thoroughly review this until next week, is that ok @ctrlc03

**Comment by ctrlc03** [comment]
> I can't thoroughly review this until next week, is that ok @ctrlc03

no rush, next week is fine :)

File: packages/backend/package.json [hunk 1]
```diff
@@ -4,16 +4,30 @@
...
     "typescript": "^5.5.4"
   },
   "dependencies": {
+    "body-parser": "^1.20.2",
     "cors": "^2.8.5",
-    "express": "^4.19.2"
+    "dotenv": "^16.4.5",
+    "ethers": "^6.13.2",
+    "express": "^4.19.2",
+    "googleapis": "^140.0.1",
```
Comment (JohnGuilding) [inline packages/backend/package.json]: Is `googleapis` needed?

**Review by JohnGuilding — APPROVED** [review]
LGTM. Tested locally and worked for me, no major comments

File: packages/backend/package.json (hunk 1 above)
Comment (ctrlc03) [reply packages/backend/package.json]: nope, will remove

**Pull request merged**